GitWorkflow/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Git Integration and CI/CD Process with SAP Data Intelligence
+
+In this project we describe a process to develop [SAP Data Intelligence (SAP DI)](https://www.sap.com/products/data-intelligence.html) solutions using Git and to build, test, and deploy them
+using [Continuous Integration and Delivery (CI/CD)](https://en.wikipedia.org/wiki/Continuous_integration) tools.
+The provided guides describe best practices and can be adapted to the specific
+needs of your development environment.
+
+The following guides and tools have been published for the following SAP DI versions and above:
+  * **SAP Data Intelligence 3.0** (on-premise edition)
+  * **SAP Data Intelligence DI:2003** (cloud edition)
+
+## Overview
+
+The environment for the Git workflow and the CI/CD process consists of the following systems and tools:
+
+* **SAP Data Intelligence**
+  * One or several development tenants
+  * A single test tenant
+  * The [System Management Command-Line Client (vctl)](https://help.sap.com/viewer/41b069490705457e9426b112a3f052bd/Cloud/en-US)
+* **Git Repository** (e.g., on GitHub or GitHub for Enterprises) to keep the [solution artifacts tracked in Git](GitWorkflow.md)
+* **Build Server** (e.g., [Jenkins](https://jenkins.io/)) to run the vctl commands of the CI/CD pipeline
+
+<img src="img/cicd_flow_di.png" alt="Development Workflow" width="900"/>
+
+The end-to-end development workflow can be summarized as follows:
+
+1. **Development**: A developer will create/update/delete graphs, operators, or dockerfiles using the SAP Data Intelligence Pipeline Modeler. All changes are visible in the user workspace of SAP Data Intelligence (`/vhome`).
+2. **Git Usage**: The developer is using the [VSCode application](vscode-app) to interact with the Git repository (i.e., `git add <graph-artifact>`, `git commit`).
```

Comment:
If all development is done in the Pipeline Modeler, then developers could also use [Github Desktop](https://desktop.github.com/) to interact with the Git repository.